package.json

```diff
@@ -50,7 +50,7 @@
     "eslint-plugin-promise": "^6.0.0",
     "eslint-plugin-security": "^1.4.0",
     "eslint-plugin-standard": "^4.1.0",
-    "firebase-tools": "^11.0.0",
+    "firebase-tools": "^11.24.1",
```

Comment:
Do not upgrade the package. If it is required, create another ticket and raise the PR for it.

Reply:
Thank you for pointing this out. I totally forgot to replace this.
I had to upgrade the firebase package because on my local the version provided in lock file was giving error while running test.
I will rectify this.